src/BloomExe/Browser.cs

```diff
@@ -29,7 +31,7 @@ public partial class Browser : UserControl
 		private string _url;
 		private XmlDocument _rootDom; // root DOM we navigate the browser to; typically a shell with other doms in iframes
 		private XmlDocument _pageEditDom; // DOM, dypically in an iframe of _rootDom, which we are editing.
-		private TempFile _tempHtmlFile;
+		private IDisposable _dependent; // Something we keep track of to dispose when the browser is disposed. Currently may be TempFile or SimulatedPageFile
```

Comment:
Maybe "_dependentContent"? In the new case it is NOT a file, just an entry in a dictionary.